ci/praktika/json.html

```diff
@@ -557,13 +557,21 @@
         widget.style.textAlign = 'left';
 
         const base_url = window.location.origin + window.location.pathname;
-
-        links.forEach(link => {
+        const items = links
+            .map((href) => {
+                const file_name = link.includes(base_url) ? "Report" : link.split('/').pop();
+                return { href, file_name };
+            })
+            .sort((a, b) =>
+                a.file_name.localeCompare(b.file_name, undefined, { numeric: true, sensitivity: 'base' })
+            )
+
+        items.forEach(({href, file_name}) => {
```

Comment:
If `href` isn't used, use `_` as a placeholder, otherwise it is confusing.

Reply:
Indeed, there was an error in the variable names. Thanks for noticing! 

Fixed